generic/tests/cfg/boot_vm_in_hugepage.cfg

```diff
@@ -5,12 +5,21 @@
     kill_vm_on_error = yes
     login_timeout = 240
     reboot_count = 25
-    slots_mem = 4
-    maxmem_mem = 32G
     setup_hugepages = yes
-    mem_devs = hugepage
-    size_mem_hugepage = 256M
-    backend_mem = memory-backend-file
-    mem-path = /mnt/kvm_hugepage
-    pre_command_noncritical = yes
-    pre_command = "echo 3 > /proc/sys/vm/drop_caches"
+    variants:
+        - default_2M_hugepage:
+            slots_mem = 4
+            maxmem_mem = 32G
+            mem_devs = hugepage
+            size_mem_hugepage = 256M
+            backend_mem = memory-backend-file
```

Comment:
Refer the comment from huangyum in https://github.com/autotest/tp-qemu/pull/1038:
"For this test, I don't think a pc-dimm is needed. Current qemu cmdline is OK."
No need to add backend_mem here ?